### DISCUSSION TITLE
Add missing "No labor" tooltip on Problems overlay

### DISCUSSION THREAD

**Discussion by Ouaz** [Keriew/augustus#544] [General]
Tiny suggestion here.

Currently, there's 5 tooltips on Problems overlay:

```
{TR_TOOLTIP_OVERLAY_PROBLEMS_STRIKE, "Workers on strike" },
{TR_TOOLTIP_OVERLAY_PROBLEMS_NO_WATER_ACCESS, "No water access" },
{TR_TOOLTIP_OVERLAY_PROBLEMS_CARTPUSHER, "Waiting for the cartpusher" },
{TR_TOOLTIP_OVERLAY_PROBLEMS_NO_RESOURCES, "No resources available" },
{TR_TOOLTIP_OVERLAY_PROBLEMS_MOTHBALLED, "Building mothballed" },
```
Buildings not having access to labor (because there's not enough available labor in the city or they are too far from houses when playing without global labour pool), are displayed on the Problems overlay, but have no tooltip.

For consistency,` {TR_TOOLTIP_OVERLAY_PROBLEMS_NO_LABOR, "No labor" }` should be added.

**Comment by crudelios** [comment]
We should add a "reduced labour: x/x workers" too.

**Comment by Keriew** [comment]
I added the one with no workers at all so far. I think there's a potential issue, that because of how workers are distributed (somewhat evenly), that if the city lacks a fair chunk of workers, then all the buildings are going to light up, kind of defeating the purpose of letting you see the issues at a glance. Opinions?

**Reply by crudelios** [reply]
That's a fair point. We could eventually add a new "employment" overlay, which would do just that.

**Reply by Keriew** [reply]
Employment overlay is a good idea.

**Comment by Ouaz** [comment]
Just tested ca2ce51, the "no labor" tooltip is not displayed for farms, raw materials buildings, and workshops (once they have resources but no labor). The "no labor" tooltip is correctly displayed for any other building type though.

**Reply by Keriew** [reply]
Good catch!

Fixed in a10a57d.
